Utils/tests/old_content_branch_test.py

```diff
@@ -64,5 +70,24 @@ def test_handle_yml__should_delete():
     Then
     - function returns False (file should be deleted)
     """
-    yml_content = get_yaml(YML_SHOULD_DELETE)
-    assert should_keep_yml_file(yml_content, TEST_VERSION, should_rewrite=False) is False
+    with open(YML_SHOULD_DELETE, 'r') as yml_file:
+
+        yml_content = ryaml.load(yml_file)
+    assert should_keep_yml_file(yml_content, TEST_VERSION) is False
+
+
+def test_edit_playbooks_directory():
+    """
+    Given
+    - A path to a tpb that should stay and its toversion field should change.
+
+    When
+    - Running edit_playbooks_directory on it.
+
+    Then
+    - Tpb should be updated with a new toversion field
+    """
+    edit_playbooks_directory(TEST_VERSION, 'Utils/tests/test_data_old_content')
+    with open(TEMP_TPB, 'r') as yml_file:
```

Comment:
lets use pytest fixture `tmp_path` instead

Reply:
why should we use tmp_path ?
its not really a temp path, ill change the name

Reply:
because we shouldnt modify actual files, it will require us to undo the changes every time we want to run the test
we should modify temp files that pytest caches over tests